app/main/views/service_settings.py

```diff
@@ -1188,6 +1189,34 @@ def edit_data_retention(service_id, data_retention_id):
     )
 
 
+@main.route("/services/<uuid:service_id>/notes", methods=['GET', 'POST'])
+@user_has_permissions('manage_service')
+def edit_service_notes(service_id):
+    form = EditServiceNotesForm()
+
+    if request.method == 'GET':
+        form.notes.data = current_service.notes
```

Comment:
You can achieve the same thing by passing `notes` as an argument to the form’s constructor:
```suggestion
    form = EditServiceNotesForm(notes=current_service.notes)
```
WTForms will make sure the `POST` data doesn’t get overridden with the existing value.